src/madsci_workcell_manager/madsci/workcell_manager/workcell_server.py

```diff
@@ -344,10 +376,31 @@ async def start_workflow(
                         workflow=wf,
                         files=files,
                     )
+
                     with state_handler.wc_state_lock():
                         state_handler.set_active_workflow(wf)
                         state_handler.enqueue_workflow(wf.workflow_id)
+
+                    # ===== WORKFLOW_START EVENT LOGGING =====
+                    logger.log(
+                        Event(
+                            event_type=EventType.WORKFLOW_START,
+                            event_data={
```

Comment:
Assign any relevant values to the workflow and then dump the workflow, rather than bespoke event data